tasklist/importer-850/src/main/java/io/camunda/tasklist/zeebeimport/v850/processors/es/ProcessZeebeRecordProcessorElasticSearch.java

```diff
@@ -115,6 +115,7 @@ private ProcessEntity createEntity(
     processEntity.setBpmnProcessId(process.getBpmnProcessId());
     processEntity.setVersion(process.getVersion());
     processEntity.setTenantId(process.getTenantId());
+    processEntity.setBpmnXml(new String(process.getResource()));
```

Comment:
Should we add the Encoding here?

`+processEntity.setBpmnXml(new String(process.getResource(), StandardCharsets.UTF_8));`

Reply:
I don't see the need, UTF-8 is set the default, do you think we should explicitly do this?

Reply:
If it is default, not necessary (Didnt know this information 💯 ) 

Thank you for clarifying this.